src/gepa/proposer/reflective_mutation/reflective_mutation.py

```diff
@@ -104,6 +113,9 @@ def propose(self, state: GEPAState) -> CandidateProposal | None:
         if self.use_wandb:
             import wandb  # type: ignore
             wandb.log({"subsample_score": sum(eval_curr.scores)}, step=i)
+        elif self.use_mlflow:
```

Comment:
Wouldn't it be better to use `self.experiment_tracker.log_metrics` after defining it, instead of having if/elif for wandb and mlflow, since that is already handled in experiment_tracker class?

Reply:
Great call, I moved the logic and propagate experiment_tracker to the downstream components.